diff --git a/node_modules/@livekit/react-native-webrtc/src/index.ts b/node_modules/@livekit/react-native-webrtc/src/index.ts
index 5a25f31..1931148 100644
--- a/node_modules/@livekit/react-native-webrtc/src/index.ts
+++ b/node_modules/@livekit/react-native-webrtc/src/index.ts
@@ -1,6 +1,9 @@
-import { NativeModules, Platform } from 'react-native';
+import { NativeModules, Platform, TurboModuleRegistry } from 'react-native';

-const WebRTCModule = NativeModules.WebRTCModule;
+const WebRTCModule =
+    NativeModules.WebRTCModule ??
+    (TurboModuleRegistry.get?.('WebRTCModule') as any) ??
+    null;

 if (WebRTCModule === null || WebRTCModule === undefined) {
     throw new Error(`WebRTC native module not found.\n${Platform.OS === 'ios' ?
